.gitignore

```
@@ -4,6 +4,7 @@
 .venv
 
 data/
+db/db

```

it should be db/data
```suggestion
db/data
```